ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -705,7 +845,7 @@ const CreatePanel = ({creatorFreeCreditsRemaining, profile, setPanelContent, wor
     <div id='compute-resources'>- Default: compute size of
       <b> {runtimeConfig.machine.cpu} CPUs</b>,
       <b> {runtimeConfig.machine.memory} GB memory</b>, and a
-      <b> {runtimeConfig.diskSize} GB disk</b>
+      <b> {runtimeConfig.diskSize} GB Persistent disk</b>
```

Comment:
lowercase "persistent"

but also, I think just saying "disk" is fine - do we need this change?

Reply:
Now we have both local disk and pd. I think renaming it might be more clearer for user to see what his change really is